src/clarity-angular/utils/focus-trap/focus-trap.directive.spec.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {ComponentFixture, TestBed} from "@angular/core/testing";
+import {Component} from "@angular/core";
+import {ClarityModule} from "../../clarity.module";
+import {FocusTrapDirective} from "./focus-trap.directive";
+import {By} from "@angular/platform-browser/";
+
+describe("InputTrap", () => {
+    let fixture: ComponentFixture<any>;
+    let compiled: any;
+    let directive: FocusTrapDirective;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [ClarityModule.forRoot()],
```

Comment:
Actually, you only want to import `ClrFocusTrapModule`. Declaring all of Clarity is overkill and will _severely_ slow down tests.